app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -77,6 +74,14 @@ class ConfigurationHeuristic(private val heuristicConfigurationData: HeuristicCo
       new HeuristicResultDetails(
```

Comment:
@shankar37 I did not understand this comment. Where should I add recommendations to the heuristic details?